style.css

```diff
@@ -0,0 +1,212 @@
+body {
+	margin: 0;
+	font-family:'Montserrat-Regular';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633355)